Content/Items/BuriedArtifacts/Accessories.WarriorKingsCrown.cs

```diff
@@ -0,0 +1,41 @@
+﻿using StarlightRiver.Content.Items.BaseTypes;
+using StarlightRiver.Core.Systems.CombatMountSystem;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StarlightRiver.Content.Items.BuriedArtifacts
+{
+	public class WarriorKingsCrown : CursedAccessory
+	{
+		public override string Texture => AssetDirectory.ArtifactItem + Name;
+
+		public WarriorKingsCrown() : base(ModContent.Request<Texture2D>(AssetDirectory.ArtifactItem + "WarriorKingsCrown").Value) { }
+
+		public override void SetStaticDefaults()
+		{
+			DisplayName.SetDefault("Warrior King's Crown");
+			Tooltip.SetDefault("Cursed : All of your summon slots are converted into +2% summoning critical strike chance per slot" +
+				" and all of your sentry slots are converted into +20% summoning damage per slot\n" +
+				"Summoning damage increased by 200%\nCombat mount cooldowns reduced by 30%");
+		}
+
+		public override void SafeUpdateAccessory(Player Player, bool hideVisual)
+		{
+			Player.GetCritChance(DamageClass.Summon) += Player.maxMinions * 2;
+			Player.GetDamage(DamageClass.Summon) += Player.maxTurrets * 0.2f;
+			Player.GetDamage(DamageClass.Summon) += 2;
+			Player.GetModPlayer<CombatMountPlayer>().combatMountCooldownMultiplier -= 0.3f;
```

Comment:
I'd recommend inversing the cooldown multiplier so that adding 0.3f is 30% faster then adding another 0.3f would take it to 60% faster. subtractive like this would make -0.3f ~42% faster and then subtracting another 0.3f would take it to ~250% faster (since 0.4 = 1/2.5). Just be careful with percentages they aren't always intuitive.

as is, if you wanted 30% faster it would be (1 - 1/1.3) which would mean you subtract ~0.23 and then if you wanted another feature to add another 30% to bring it to 60% that one would be (1 - (1/1.3 - 1/1.6)) so it would have to subtract 0.145 to bring it to 60% faster (since 0.625 = 1/1.6)